docs/IN/69/IN69-verifications-recurrentes-de-la-pki.md

```diff
@@ -218,24 +218,26 @@ Finally, force an unseal share rotation by executing the following script. It sh
 
 An unseal share rotation should be run also on the online vault.
 
-### Valid certificates can be issued by the online CA during at least 1 more year
+### Valid certificates can be issued by the online CA during the next 18 months to come
 
 The online CA that signs device certificates should be able to perform valid signatures until the next recurrent check (plus some margin), that is to say, for the next 18 months.
 
 #### Test
 
-Take the current date, and add 18 months. Then add again the duration of devices certificates. The result is a date *D<sub>min</sub>*.
+Take the current date now, it will be called *D<sub>now</sub>*. Add 18 months to *D<sub>now</sub>*, then add again the duration of devices certificates. The result is a date *D<sub>min</sub>*.
 
-In order to get the expiry date of the online CA currently in use to sign devices, run the following command against the **online vault**:
+In order to get the expiry date of the online CA currently in use to sign devices, run the following command against the **online vault** after being logged in to the vault using an account with **admin** privileges:
 ```bash
 vault read -format=json devices_pki/issuer/default/json | jq -r '.data.certificate' | openssl x509 -noout -text
 ```
 
-The expiry date will be called *D<sub>expiry</sub>*, it can be found in the `Validity` attributes displayed by the command above (`Not Before` and `Not After`).
+The start of validity date will be called *D<sub>start</sub>*, it can be found in the `Validity`'s `Not Before` attribute displayed by the command above.
+The expiry date will be called *D<sub>expiry</sub>*, it can be found in the `Validity`'s `Not After` attribute displayed by the command above.
 
-The following statement should be true *D<sub>expiry</sub>*>*D<sub>min</sub>*.
+The following statement should be true:
+*D<sub>start</sub>*>*D<sub>now</sub>* **and** *D<sub>expiry</sub>*>*D<sub>min</sub>*.
```

Comment:
`start > now` ? Isn't it the opposite? (my head might not be wired correctly at the moment 😆 )

Reply:
yeah, if the online CA currently is in use to sign devices, it should be < now

Reply:
Yes, that's right. Fixed in d9318ead3cb09b412d0ed6f2fead3b8c1a5a7a3a